packages/local-runner/python/deepnote_pipeline.py

```
@@ -0,0 +1,756 @@
+                    notify("skipped", step.id)
+                    continue
+
+                futures = []
+                for instance_id, scope in admitted:
+                    notify("started", instance_id)
+                    futures.append(
+                        pool.submit(api.run_notebook, step.notebook_id, resolve_value(step.inputs, scope))
+                    )
+
+                runs = []
+                for (instance_id, _scope), future in zip(admitted, futures):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)